src/main/java/org/whispersystems/textsecuregcm/auth/AuthorizationTokenGenerator.java

```diff
@@ -17,23 +18,26 @@
 
   private final Logger logger = LoggerFactory.getLogger(AuthorizationTokenGenerator.class);
 
-  private final byte[] key;
+  private final byte[]           key;
+  private final Optional<byte[]> userIdKey;
```

Comment:
Do we need this to be optional?

Reply:
done in 3c56173315d46d8fe4fc7bd1834f61dda0543b03